dev/projects.md

```diff
@@ -17,18 +17,22 @@
 
 ## Creating a Feature Branch
 
+**Note:** You do not have to do this if you are the only one working on the branch. Just
+create your own local branch and create a PR there.
+
 1. Make sure you are on the `develop` branch in the [Github
    repository](https://github.com/jhuopensource/semesterly). Click on the dropdown where
    you can search or create a branch, and create a branch from here called
    `feature/your-feature-name`.
 
-2. In your editor now, `git checkout feature/your-feature-name` to switch to the branch
-   you just created.
+2. In your editor now, `git fetch` and `git checkout feature/your-feature-name` to
+   switch to the branch you just created.
```

Comment:
Can you give an example? I'm confused why it would be in the pattern `feature/feature-name`. It seems redundant.

Reply:
I guess I see the example from the branch of this PR (but it should be written in this document. However, I'm concerned this equates to a large number of branches. Unless the solution is to delete all the branches once they're merged?

Reply:
This is the new naming convention for branches
https://semesterly-v2.readthedocs.io/en/latest/contributing.html

I don't see how having a large number of branches is a problem; they are really easy to clean.